src/components/Synthetics/TVChart/TVChartHeader.tsx

```diff
@@ -358,27 +370,87 @@ function TVChartHeaderInfoDesktop() {
     low24,
   ]);
 
+  const scrollTo = useCallback(
+    (dir: 1 | -1) => () => {
```

Comment:
I think, this memoization doesn't work as intended. Would be better to try something like:
```
const scrollToUp = useCallback(() => scrollTo(-1), [scrollTo]);
const scrollToDown = useCallback(() => scrollTo(1), [scrollTo]);
```